### DISCUSSION TITLE
What are profiles

### DISCUSSION THREAD

**Discussion by CoolTrainerEX** [gorilla-devs/ferium#375] [Q&A] [answered]
What are profiles and what does having multiple profiles and switching between them do? Are they meant to change the active mods between them? Why when I performed `ferium profile switch` the `mods` folder did not change?

**Comment by theRookieCoder** [comment, marked as answer]
Profiles are separate lists of mods for you to keep track of. Ideally, you would point each profile to a different directory and link them with different profiles in your launcher (hence the name). Multiple profiles aren't really meant to point to the same directory, but I have yet to implement any sort of warning when something like that is done.
